twenty-first/src/util_types/mmr/mmr_accumulator.rs

```diff
@@ -1022,11 +1037,13 @@ mod accumulator_mmr_tests {
         }
     }
 
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
     #[test]
     fn computing_mmr_root_for_no_leafs_produces_some_digest() {
         MmrAccumulator::new_from_leafs(vec![]).bag_peaks();
     }
 
+    // note: snapshot not supported for wasm32 target.
```

Comment:
Too bad. I think it's better to keep the test and to remove the dependency on `insta`. To be frank, we don't really need the functionality provided by `insta` anyway, as we're only asserting on `u64`s and the occasional `Digest` with it. To me, it's much more important to have the snapshot tests also in wasm than to communicate that the assertion is a snapshot test.